zebra-rpc/src/methods.rs

```diff
@@ -3822,7 +3872,25 @@ impl Default for GetRawTransactionResponse {
 }
 
 /// Response to a `getaddressutxos` RPC request.
-pub type GetAddressUtxosResponse = Vec<Utxo>;
+#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(untagged)]
+pub enum GetAddressUtxosResponse {
+    /// Response when `chainInfo` is false or not provided.
+    ChainInfoFalse(Vec<Utxo>),
+    /// Response when `chainInfo` is true.
+    ChainInfoTrue(GetAddressUtxosResponseObject),
```

Comment:
Optional/Nitpick/Subjective: Could we make this `Utxos` and `UtxosAndChainInfo`? Or `List` and `Object`?

```suggestion
    /// Response when `chainInfo` is false or not provided.
    Utxos(Vec<Utxo>),
    /// Response when `chainInfo` is true.
    UtxosAndChainInfo(GetAddressUtxosResponseObject),
```

Reply:
Done in ceea7a9c8